lib/plugins/backup.rb

```diff
@@ -3,8 +3,8 @@
 require_relative '../moonshot/creds_helper'
 
 module Moonshot::Plugins # rubocop:disable Documentation
-  # Monkeypatching TarWriter for our
-  # needs to improve clarity
+  # Defining new method for the ease of use in this
```

Comment:
If there is no original method, why monkey-patch at all? Can you just use a private method in our own namespace?